otterdog/eclipse-keypop.jsonnet

```diff
@@ -105,6 +105,71 @@ orgs.newOrg('eclipse-keypop') {
         },
       ],
     },
+    orgs.newRepo('keypop-calypso-certificate-cpp-api') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Eclipse Keypop project repository containing a C++ implementation of the 'Calypso Certificate API' standardized by the Calypso Networks association for ticketing terminal processing smart card",
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_projects: false,
+      has_wiki: false,
+      homepage: "https://keypop.org/",
+      topics+: [
+        "api",
+        "component",
+        "cpp",
+        "keypop"
+      ],
+      web_commit_signoff_required: false,
+      workflows+: {
+        actions_can_approve_pull_request_reviews: false,
+      },
+      environments: [
+        orgs.newEnvironment('github-pages') {
+          branch_policies+: [
+            "gh-pages"
+          ],
+          deployment_branch_policy: "selected",
+        },
+      ],
+    },
+    orgs.newRepo('keypop-calypso-certificate-java-api') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Eclipse Keypop project repository containing a Java implementation of the 'Calypso Certificate API' standardized by the Calypso Networks association for ticketing terminal processing smart card",
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_projects: false,
+      has_wiki: false,
+      homepage: "https://keypop.org/",
+      topics+: [
+        "api",
+        "component",
+        "java",
+        "keypop"
+      ],
+      web_commit_signoff_required: false,
+      workflows+: {
+        actions_can_approve_pull_request_reviews: false,
+      },
+      secrets: [
+        orgs.newRepoSecret('SONAR_TOKEN') {
```

Comment:
I double checked our credential store and we currently do not know the sonar token that is used as secret for other repos as well. Do you happen to have access to that token and can share it with us? Then we can reference it in the configuration as well and it gets provisioned.

Sharing of secrets with EF: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/wikis/Sharing-secrets

Reply:
Thank you for raising this issue.

When Keypop was created in an independent GitHub organization, we didn't think about this matter (our other Keyple project was then an Eclipse organization project on both the GitHub and SonarCloud sides, so the SONAR token had to be created by someone at Eclipse).

I'm not sure by whom and how this issue is managed on the SonarCloud side when the organization is independent.

Have you come across a similar case with another project using SonarCloud that could serve as a model for what needs to be done?

Thanks for your feedback.

Reply:
ok thank you for your answer, that helps me to understand how this was setup. I found a ticket in the helpdesk, so we should be able to create a new token and provision that. Will come back to you.

Reply:
You're right, Keypop being an interface repository, SonarCloud is useless.
I'm having my pull request completed to remove the SONAR configuration.
Thanks.

Reply:
no you misunderstood me, afaict sonar integration is done via their GitHub app that is configured for the keypop organization. You just have to select there which repos should be configured for sonar. So the sonar token itself is useless as the integration is done via the app instead of a workflow that would use the sonar token. If you agree, after creating the repo, I can enable the sonarcloud integration for it afterwards.

Reply:
I understand. However, I can confirm that we have no need to use SonarCloud for these interfaces, which really don't contain any code.